frappe/utils/data.py

```diff
@@ -986,8 +986,7 @@ def _expand_relative_urls(match):
 	html = re.sub('(href|src){1}([\s]*=[\s]*[\'"]?)((?!http)[^\'" >]+)([\'"]?)', _expand_relative_urls, html)
 
 	# background-image: url('/assets/...')
-	html = re.sub('(:[\s]?url)(\([\'"]?)([^\)]*)([\'"]?\))', _expand_relative_urls, html)
-
+	html = re.sub('(:[\s]?url)(\([\'"]?)((?!http)[^\'" >]+)([\'"]?)', _expand_relative_urls, html)
```

Comment:
```suggestion
	html = re.sub('(:[\s]?url)(\([\'"]?)((?!http)[^\'" >]+)([\'"]?\))', _expand_relative_urls, html)
```

The last bracket is used in a condition to add `!important` (A small hack for wkhtmltopdf)

https://github.com/frappe/frappe/blob/8e9830ebdabcb2041a7b7d559a9dbac5e3807aa4/frappe/utils/data.py#L980-L982